contents/docs/data/events.mdx

```diff
@@ -63,4 +63,58 @@ If something has been detected incorrectly, you can manually change the type by
     imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/edit_property_dark_23507c2df2.png"
     alt="Edit properties" 
     classes="rounded"
-/>
\ No newline at end of file
+/>
+
+## Anonymous vs identified events
+
+By default, PostHog captures identified events. This means all events create a [person profile](/docs/data/person) and set [person properties](/docs/data/person-properties). 
```

Comment:
We are changing this default. All snippets contain "identified_only" so I probs wouldn't mention this default now, just mention the config.

Reply:
For right now, I will say that identified events is the default because it is even if we add person_profiles to all the snippets. This is especially true on the backend. Will change when the default changes.